File: Linguagem C/soma_linhas.c
#include <stdio.h>
#include <stdlib.h>

int main() {


    int c,l,i,j;

    printf("Qual a quantidade de linhas da matriz? ");
    scanf("%d", &l);
    printf("Qual a quantidade de colunas da matriz? ");
    scanf("%d", &c);
    double mat[l][c];
    double vet[l] , soma;

    soma = 0;
    for(i = 0; i < l; i++) {
        printf("Digite os elementos da %da. linha: \n", i+1);
        for(j = 0; j < c; j++) {
            scanf("%lf", &mat[i][j]);
            soma = soma + mat[i][j];
        }
        vet[i] = soma;
        soma = 0;
    }
    printf("\n");

    printf("VETOR GERADO: \n");
    for(i = 0; i < l; i++) {
        printf("%.1lf \n", vet[i]);
    }
    return 0;
}

File: Linguagem Python/EXs - Matrizes PY/soma_linhas.py
L = int(input("Qual a quantidade de linhas da matriz? "))
C = int(input("Qual a quantidade de colunas da matriz? "))

mat = [[0 for x in range(C)] for x in range(L)]
for i in range(0, L):
    print(f"Digite os elementos da {i+1}a. linha: ")
    for j in range(0, C):
        mat[i][j] = int(input())

soma: [float] = [0 for x in range(L)]
for i in range(0, L):
    for j in range(0, C):
        soma[i] = soma[i] + mat[i][j]

print("Vetor Gerado:")
for i in range(0, L):
    print(f"{soma[i]:.1f}")